plugins/core/db/index.js

```diff
@@ -1,11 +1,63 @@
-const nano = require('nano');
-const Boom = require('boom');
+const nano = require("nano");
+const Boom = require("boom");
+
+function getFilters(queryParameters) {
```

Comment:
please name this function `getSearchFilters` as this is only covering the fixed set of search query parameters we support and is not a generic function to generate query parameters to filters for mango.